.travis.yml

```diff
@@ -42,10 +47,15 @@ jobs:
         # Substitute build string
         - sed -i.bak "s/buildInfo = \"\"/buildInfo = \"$TRAVIS_BRANCH-$TRAVIS_COMMIT\"/" tapdance/logger.go
         - mkdir -p $GOPATH/src/github.com/Psiphon-Labs
+        # Get Specific Psiphon Core Library Release
         - git clone https://github.com/Psiphon-Labs/psiphon-tunnel-core.git $GOPATH/src/github.com/Psiphon-Labs/psiphon-tunnel-core
+        - (cd $GOPATH/src/github.com/Psiphon-Labs/psiphon-tunnel-core && git checkout $PSIPHON_CORE_VERSION -b build-refraction-networking)
```

Comment:
Same only question as in #21: why did we switch the branch? Is Psiphon going to maintain that branch?